nunca-ouvi-falar-de-rust/index.en.html

```diff
@@ -223,14 +227,14 @@ <h2 id="org26774a6">"Why did they choose to create a low-level language?"</h2>
 </section>
 <section >
 <p>
-Memory management without a garbage collector
+Memory management without a Garbage Collector
```

Comment:
Capitalized? Any special reason for this?

Reply:
I'm not sure, I felt like this is a named concept, so we should capitalize it? Would it be better to make it lower cased?